### DISCUSSION TITLE
Moved from Standard £0.29 per unit to https://octopus.energy/smart/cosy-octopus/

### DISCUSSION THREAD

**Discussion by sjiuk** [BottlecapDave/HomeAssistant-OctopusEnergy#328] [Q&A] [answered]
Hi I have moved from Standard £0.29 per unit to https://octopus.energy/smart/cosy-octopus/ and I am still seeing the standard rates and not the Cosy rates in Home Assistant, any ideas?

**Comment by BottlecapDave** [comment, marked as answer]
Hi there. When did you switch and when does it become active? It sometimes takes a few days for the API to update and reflect changes. Some people have also reported that reloading the integration can force changes through.

**Reply by sjiuk** [reply]
Thank you. I am new to HACS and I found I was a version behind, after updating and restarting Home Assistant all is working.
